app/decorator.go

```diff
@@ -0,0 +1,29 @@
+package app
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type forceCheckTxProxyDecorator struct {
+	d sdk.AnteDecorator
+}
+
+func newForceCheckTxProxyDecorator(d sdk.AnteDecorator) forceCheckTxProxyDecorator {
```

Comment:
i think this file is duplicated in `cosmos/decorator`.
let's remove the one in `cosmos/decorator`.